documentation/writing_yaml_plans.md

```diff
@@ -124,6 +124,34 @@ prints the object as a string.
 For information on printing a step result with `message`, see [Debugging
 plans](#debugging-plans).
 
+
+### Verbose step
+
+Use a `verbose` step to print a message in verbose mode. The step prints a message to standard
+out (stdout) when using the `human` output format, and prints to standard error
+(stderr) when using the `json` output format. 
+
+Verbose steps support the following keys:
+
+| Key | Type | Description | Required |
+| --- | --- | --- | --- |
+| `verbose` | `String` | The message to print. | ✓ |
```

Comment:
```suggestion
| `verbose` | `Any` | The message to print. | ✓ |
```

Reply:
We should update this for `message` in this doc too - originally this function *only* accepted Strings, but we [added support for printing any object](https://github.com/puppetlabs/bolt/pull/2055/files) and just missed updating these docs.